replit/nixmodules
# remove TS LSP from SvelteKit

Why
===

Turns out Svelte's LSP already has JS/TS file support

What changed
============

removed TS LSP from SvelteKit

Test plan
=========

- open foo.js
- enter invalid expression
- only svelte LSP should report the error

Rollout
=======

_Describe any procedures or requirements needed to roll this out safely (or check the box below)_
